tasks/server_keys.yml

```diff
@@ -154,14 +154,14 @@
 - name: Install crl-cron script
   template:
     src: crl-cron.sh.j2
-    dest: "{{ openvpn_base_dir }}/crl-cron.sh"
+    dest: "{{ openvpn_base_dir }}/crl-cron_{{ openvpn_config_file }}.sh"
```

Comment:
I'm not sure why you need to change this.

What is your use case?

Reply:
When `openvpn_key_dir` variable is set, this file contains different path for each openvpn instance.

My case is - few playbooks with different users/routes, different `openvpn_server_network`, `openvpn_server_netmask`, `openvpn_key_dir`, `openvpn_port` and other variables applied to one server.

Reply:
So multiple OpenVPN servers on the same server. Hadn't thought of that.

How about you just change it to use the `{{ openvpn_key_dir }}` instead of renaming it within `{{ openvpn_base_dir }}`, then update the cron line that references it: https://github.com/kyl191/ansible-role-openvpn/blob/master/tasks/server_keys.yml#L181